src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,180 @@
+// Avatar Component
+
+import { cn } from "@/utils";
+import { cva, VariantProps } from "class-variance-authority";
+import React, { ComponentProps } from "react";
+
+// Avatar
+const AvatarStyles = cva(
```

Comment:
Let's move styling to last or create a separate file. From a personal experience, most of the time, developers will open this file to see the component structure. Thoughts?